collector/settings.mdx

```diff
@@ -305,6 +305,16 @@ Note that the `aws_endpoint_*` settings are only relevant if you are using custo
       <td>If specified, and api_system_scope (see below) is not specified, this is prepended to the auto-generated system scope (optional, can be used
       to, e.g., differentiate staging from production)</td>
     </tr>
+    <tr>
+      <td>db_use_iam_auth (<code>DB_USE_IAM_AUTH</code>)</td>
```

Comment:
I noticed this is the only AWS-specific setting that does not start with `aws_`. Do we want to rename it prior to documenting it?

Reply:
That was intentional, Google also calls their implementation "IAM authentication", and so it seemed like we could use this as a more general term. See https://github.com/pganalyze/collector/pull/368#discussion_r1087387656

Reply:
Okay. I think it maybe would have made more sense to have parallel `aws_use_iam_auth`, `gcp_use_iam_auth`, etc., but this is reasonable. I imagine we still want to keep it in the AWS section, though, at least for now?

Reply:
Oh, just saw your other comment. I'll keep it here.